x-pack/plugins/session_view/kibana.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": "sessionView",
+  "version": "8.0.0",
```

Comment:
Is this the right version you want? I don't think this is meant to be in sync with the version of Kibana if that's why you set it to 8? Though, I will say I'm not sure where it's actually used

Reply:
Ha yea, I would say it should start at v1.0.0, but that usually means the feature is complete, which it is not :) 0.1.0 maybe a good place to start, and when we have an MVP we could bump to 1.0.0? So subjective :)

Reply:
Yea, not really a big deal, just wanted to note. I don't think the security solution version has been updated in.....ever 😂